.circleci/config.yml

```diff
@@ -79,10 +79,10 @@ aliases:
 version: 2.1
 
 jobs:
-  bootstrap_latest:
+  bootstrap_v12:
     executor:
       name: node
-      image: '11' # temporarily set to v11 while Firebase fixes v12 support
+      image: "12"
```

Comment:
Firebase tools support Node.js v12 now, but not the latest version, ie. Node.js v14 yet. They also deprecated Node.js v8